cross-edits-analysis/README.md

```diff
@@ -1 +1,23 @@
-This dir contains anlaysis parts of https://github.com/google/wikiloop-battlefield/issues/245
+# Google_Cross_Edit_Pattern_Exploration
+This repository is created to host exploratory code for the Wikipedia Revision Dataset, as part of the Cross Edit Pattern Detection Project. <br />
+This repository will be migrated to a Google corporate repository on June 10th, 2020. <br />
+Author: Haoran Fei (haoranfei@google.com) <br />
+Date: June 8th, 2020 <br />
+
+# Open-Source Dependencies and Licensing
```

Comment:
If you are merely calling them as library, you don't have to add licensing for Python3, Pandas or Matplotlib. You only do when you are distributing their source code.

Reply:
This is mostly for my own record.